### ISSUE TITLE
'High Contrast Black' theme Avatar color contrast

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#997]
**Describe the bug**
Icons in the Avatar component do not have a contrast ratio of 3:1 or above.
Icons that can be used to understand the content need a minimum contrast ratio of 3:1 according to this WCAG 2.1 specification: https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html 

> What version of the Fundamental Library Styles are you using?
v0.9.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/avatar.html
2. Check Icon color contrast

**Expected behavior**
Icon in the Avatar component should have a color contrast ratio of 3:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81631210-32954700-93bc-11ea-85c3-c5f3c1239b0d.png)
![image](https://user-images.githubusercontent.com/43764832/81631271-55bff680-93bc-11ea-855b-f7071ef84a9b.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of https://github.com/SAP/fundamental-styles/pull/1789
![image](https://user-images.githubusercontent.com/21978807/96763297-6ecacb80-138e-11eb-80e4-d5f2439d3e0e.png)
